build.sh

```diff
@@ -1,101 +1,77 @@
 #!/usr/bin/env bash
-
-##########################################################################
-# This is the Cake bootstrapper script for Linux and OS X.
-# This file was downloaded from https://github.com/cake-build/resources
-# Feel free to change this file to fit your needs.
-##########################################################################
-
-# Define directories.
+# Define varibles
```

Comment:
can you provide a link to the file which you used as an example here? You probably used some other link than this: https://github.com/cake-build/resources/blob/master/dotnet-framework/build.sh?

Reply:
The diff I made was to make sure we had multiple .NET cores installed if they didn't exist, since the Cake build scripts assume that only 1 version of .NET core would be installed and no more. Now with https://jira.mongodb.org/browse/BUILD-12367 finished this is no longer needed and is removed.

Reply:
Actually without this change on my latest run it does fail, so I think these are still needed.

Error is `The framework 'Microsoft.NETCore.App', version '2.1.0' was not found.`

Reply:
This file also comes directly from the Cake repo. The only thing I modified in this case is in the comment here: https://github.com/durran/mongo-csharp-driver/pull/2/files/a9f97448fe0aee826bed2a3ca9f443807e04c978#r523392217